moj-node/server/routes/home.js

```diff
@@ -42,6 +42,18 @@ module.exports = function Home({
         userName: path(['name'], userDetails),
       };
 
+      const popularTopics = {
```

Comment:
Moved to code rather than the template. Easier to see, don't expect variables to only be in templates.